Sources/PackageCollections/Storage/SQLitePackageCollectionsStorage.swift

```diff
@@ -816,35 +850,25 @@ final class SQLitePackageCollectionsStorage: PackageCollectionsStorage, Closable
             return db
         }
 
+        // FIXME: workaround linux sqlite concurrency issues causing CI failures
```

Comment:
this was a fix we did in manifest loader (which also uses SQLite) to address races in SQLite getting locked up, so I wanted to bring it here too